portal-bridge/src/types/mode.rs

```diff
@@ -155,6 +159,30 @@ impl FromStr for ModeType {
     }
 }
 
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub enum FourFoursModeType {
+    Random,
+    Epoch(u64),
```

Comment:
*nitpick* imo, `Single(u64)` here is a bit more intuitive. `Single` when compared to `Random` is fairly simple to understand. I think it's safe to assume that readers will understand that the `u64` is in reference to the epoch number, though it's worth leaving a short comment... `// Gossips a single epoch`

We also use `"single"` in the history bridge, so it's a bit more consistent